content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products.rst

```diff
@@ -28,45 +30,60 @@ with each product line containing the following information:
+  might change the available quantity once fulfilled.
 - :guilabel:`On Hand Quantity`: the quantity of the product currently recorded in the database.
 - :guilabel:`UoM`: the *unit of measure* in which the product is measured. Unless otherwise
   specified (i.e., in :guilabel:`Pounds` or :guilabel:`Ounces`), the default :abbr:`UoM (Unit of
   Measure)` is :guilabel:`Units`.
+- :guilabel:`Accounting Date`: the date on which the adjustments will be accounted in the Odoo
+  *Accounting* app.
 - :guilabel:`Counted Quantity`: the real quantity counted during an inventory count. This field is
   left blank by default, but can be changed, depending on if it matches the :guilabel:`On Hand
   Quantity` or not.
 - :guilabel:`Difference`: the difference between the :guilabel:`On Hand Quantity` and
   :guilabel:`Counted Quantity`, once an inventory adjustment is made. The difference is
   automatically calculated after every inventory adjustment.
 - :guilabel:`Scheduled Date`: the date at which a count should be made. If not otherwise specified,
-  this date will default to the 31st of December of the current year.
+  this date defaults to the 31st of December of the current year.
 - :guilabel:`User`: the person assigned to the count in the database. This can either be the person
   physically counting the inventory, or applying the count in the database.
-- :guilabel:`Product Category`: the category assigned internally to a specific product. Unless
-  otherwise specified (i.e., as :guilabel:`Consumable` or :guilabel:`Rental`), the default *Product
-  Category* is set to :guilabel:`All`.
-- :guilabel:`Available Quantity`: the quantity of a specific product that is currently available,
-  based on any outstanding/uncompleted sales orders, purchase orders, or manufacturing orders that
-  might change the available quantity once fulfilled.
-- :guilabel:`Accounting Date`: the date on which the adjustments will be accounted in the Odoo
-  *Accounting* app.
 - :guilabel:`Company`: the company whose database these inventory adjustments are being made on. The
   company is listed in the top right corner of the database, next to the user currently logged in.
 
 .. tip::
-   Some columns are hidden by default. To reveal these columns, click the :guilabel:`additional
-   options` button (three dots icon) to the far right of the form's top row, and select any desired
-   column to reveal by clicking the checkbox next to that option.
+   Some columns are hidden by default. To reveal these columns, click the
+   :icon:`oi-settings-adjust` :guilabel:`(settings)` icon to the far right of the form's top row,
+   and select any desired column to reveal by clicking the checkbox next to that option.
+
+.. seealso::
+   - :doc:`Product type <../../product_management/configure/type>`
+   - :doc:`Product tracking <../../product_management/product_tracking>`, including
+      using lots, serial numbers, and expiration dates
```

Comment:
Something weird is going on with the formatting :cry: (see screenshot below)

Let's try making all of the text a link. But, if you have better ideas to make the text look good, Liz, feel free to try! 

![image](https://github.com/user-attachments/assets/b2309ae3-90da-4002-ae2a-59678d625df0)

```suggestion
   - :doc:`Product tracking, including using lots, serial numbers, and expiration dates <../../product_management/product_tracking>`
```

Or, changing the meaning a little but shortening the link text?
```suggestion
   - :doc:`Tracking lots, serials, and expiration dates <../../product_management/product_tracking>`
```